src/components/LandingFooter/LandingFooter.css

```diff
@@ -0,0 +1,11 @@
+footer {
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  margin-bottom: 2rem;
+}
+
+.LandingFooter {
+  color: white;
```

Comment:
@StephenAraka  Just for clarity 
Why do you create css for an element `footer` and its class `LandingFooter`?
- this will cause any code references of footer to take up this css.
- so is it right to do that?
Just inquiring here`

Reply:
The class name 'LandingFooter' helps to differentiate this particular footer from another footer that may exist.
This way, their styles may not class.
It will have no issues.

Reply:
Kale Sebo